client/idrepo/enduser/src/main/resources/org/apache/syncope/client/enduser/SyncopeEnduserApplication.properties

```diff
@@ -78,6 +78,7 @@ before=Before
 after=After
 
 captcha_error=Captcha entered does not match
+blank_security_info=Security question and/or answer is/are blank
```

Comment:
Is this key not needed in other languages?

Reply:
yes, sure. Going to add also to other langauges